docs/kratos/mfa/15_totp.mdx

```diff
@@ -67,15 +67,25 @@ In this example, the user identifier (email - `alice@example.com`) is used. To l
 2. In the **TOTP Authenticator App** section, use the switch to enable TOTP.
 3. Define the name of the requesting party in the **Display Name**. The users see this name in the TOTP application when getting a
    one-time password.
-4. Click **Save** to finish.
+4. Click **Save**.
 
-```mdx-code-block
-<BrowserWindow url="https://console.ory.sh/">
+   ```mdx-code-block
+   <BrowserWindow url="https://console.ory.sh/">
 
-![TOTP setup in Ory Console](../_static/mfa/3.png)
+   ![TOTP setup in Ory Console](../_static/mfa/3.png)
 
-</BrowserWindow>
-```
+   </BrowserWindow>
+   ```
+
+5. After TOTP has been enabled for the Ory Network project, each user is responsible for enabling TOTP in their own account by
+   pairing their account with a TOTP app.
+
+   :::note
+
+   In the Ory Account Experience, users are **not** automatically prompted to enable TOTP on their account. A user can enable TOTP
+   in their account by signing in and going to the **Authenticator App** section of the **Account Settings** page.
```

Comment:
```suggestion
  In the Ory Account Experience, users are not automatically prompted to enable TOTP for their accounts. To enable TOTP, users must sign in to their accounts and go to the **Authenticator App** section of the **Account Settings** page.
```

Re-commenting as this hasn't been implemented.

Reply:
Ah! Sorry, somehow messed this up. Committing now...